protocol/messenger_communities.go

```diff
@@ -4157,7 +4157,7 @@ func (m *Messenger) RequestImportDiscordChannel(request *requests.ImportDiscordC
 
 				communityChat := &protobuf.CommunityChat{
 					Permissions: &protobuf.CommunityPermissions{
-						Access: protobuf.CommunityPermissions_NO_MEMBERSHIP,
+						Access: protobuf.CommunityPermissions_AUTO_ACCEPT,
```

Comment:
mmm... and where is PR with renaming?

Reply:
Here: https://github.com/status-im/status-go/pull/4207

Reply:
found it